starter-code/index.js

```diff
@@ -1,11 +1,56 @@
 class SortedList {
-    constructor() {}
-    add(item) {}
-    get(pos) {}
-    max() {}
-    min() {}
-    average() {}
-    sum() {}
+    constructor() {
+      this.items = [];
+      this.length = 0;
+    }
+
+    add(item) {
+      this.items.push(item);
+      this.items.sort((a,b) => a - b)
+      this.length +=1
```

Comment:
Funciona com "+=1", mas seria uma melhor prática blindar seu código e fazer this.length = this.items.length.
Assim se alguém manipular o this.items sem usar o método (dando splice, shift etc), no momento que der um add(), o length ficaria correto.